sci-calculators/bc-gh/bc-gh-6.2.2.ebuild

```
@@ -0,0 +1,45 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="
+	https://git.yzena.com/gavin/bc/
+	https://github.com/gavinhoward/bc/
+"
+SRC_URI="
+	https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz
+"
+S="${WORKDIR}/bc-${PV}"
```

Let's use the traditional Gentoo `MY_P` ;-).

```suggestion
MY_P="bc-${PV}"
DESCRIPTION="Implementation of POSIX bc with GNU extensions"
HOMEPAGE="
	https://git.yzena.com/gavin/bc/
	https://github.com/gavinhoward/bc/
"
SRC_URI="
	https://github.com/gavinhoward/bc/releases/download/${PV}/${MY_P}.tar.xz
"
S=${WORKDIR}/${MY_P}
```